src/i18n.js

```diff
@@ -284,20 +284,29 @@ i18n
           },
           project: {
             accessDeniedNoAuthModal: {
+              embedded: {
+                text: "Visit the Projects site for cool project ideas",
```

Comment:
🌱 no action needed on this but just thought we'll need to restructure this a little at some stage to reduce repetition. I can see this visit project site having a wider use case but there's bound to be more and translations is a whole separate block of work 👍

Reply:
good call :thumb_up: